bears/rest/RSTcheckBear.py

```diff
@@ -1,8 +1,20 @@
+import re
+
 from coalib.bearlib.abstractions.Linter import linter
 from dependency_management.requirements.PipRequirement import PipRequirement
 from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
 
+def _ignore_unknown(errors):
```

Comment:
@Makman2 earlier I was calling this function from inside create arguments. But that seemed to be a wrong way. So, removed it .